kafka-init/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG KAFKA_VERSION
+ARG SCALA_VERSION
+
+FROM monasca/kafka:${KAFKA_VERSION:-"0.9.0.1"}-${SCALA_VERSION:-"2.11"}
```

Comment:
Replaced that with single ARG. Basically there's no point in having multiple variables as we inherit from the ```monasca/kafka``` image here. Should be much simpler now and easier to parse.